static/app/views/settings/components/teamSelect.tsx

```diff
@@ -92,6 +92,41 @@ function TeamSelect({
   const {teams, onSearch, fetching} = useTeams();
   const {orgRoleList, teamRoleList} = organization;
 
+  const slugsToFilter =
+    selectedTeams?.map(tm => tm.slug) || selectedTeamRoles?.map(tm => tm.teamSlug) || [];
+
+  // determine if adding a team changes the minimum team role
+  const getTopOrgRole = () => {
+    const teamsWithOrgRole = teams.filter(
+      team => slugsToFilter.some(slug => slug === team.slug) && team.orgRole !== null
```

Comment:
if slugsToFind is an array of slugs, could you just use `.includes` instead of `some`?

Reply:
oh good point. i was just reusing code from below